### PR TITLE
High priority effects

### PR DESCRIPTION
**Pull request by tavinathanson**
More gross code. I know it's gross.

### CODE REVIEW

**Comment by hammer** [comment]
Why are we doing so much in __init__? Why not pull out into a separate source file and just import that in init?

**Comment by tavinathanson** [comment]
@hammer Totally. It was just a starting point that I dropped into an `__init__.py` file.

File: utopia/__init__.py [hunk 1]
```diff
@@ -8,6 +9,7 @@ def predict(variant_collection,
             predictor_type='netmhcpan',
             epitope_lengths=[9],
             mutation_window_size=None,
+            priority_effects_only=False,
```
Comment (iskandr) [inline utopia/__init__.py]: Not sure about this name: "priority", which priority?

**Comment by iskandr** [comment]
Other than the naming, LGTM. Well, sort of, we have to dramatically rearrange this module. I guess that will follow on the heels of a `mhctools` cleanup.

File: utopia/__init__.py (hunk 1 above)
Comment (tavinathanson) [inline utopia/__init__.py]: What name do you think would be better?

**Comment by tavinathanson** [comment]
Cool, and I'll leave the new file creation to your PR.

File: utopia/__init__.py (hunk 1 above)
Comment (tavinathanson) [inline utopia/__init__.py]: Going to go ahead and merge so we can merge your PR.

**Pull request merged**